assets/dev/js/frontend/handlers/container/grid-container.js

```diff
@@ -221,4 +229,23 @@ export default class GridOutline extends elementorModules.frontend.handlers.Base
 		window.removeEventListener( 'elementor/editor/element-rendered', this.lifecycleChangeListener );
 		window.removeEventListener( 'elementor/editor/element-destroyed', this.lifecycleChangeListener );
 	}
+
+	updateEmptyViewHeight() {
+		if ( this.shouldUpdateEmptyViewHeight() ) {
+			const { emptyView } = this.elements,
+				currentDevice = elementor.channels.deviceMode.request( 'currentMode' ),
+				elementSettings = this.getElementSettings(),
+				gridRows = 'desktop' === currentDevice ? elementSettings.grid_rows_grid : elementSettings.grid_rows_grid + '_' + currentDevice;
```

Comment:
@davseve What if the current device value is empty? I believe you should use `elementorFrontend.utils.controls.getResponsiveControlValue()`

Reply:
I don't need the value, I only to know if the control state is `custom` or `fr`